src/rf/rf/settings/base.py

```diff
@@ -277,15 +275,15 @@ def get_env_setting(setting):
 WSGI_APPLICATION = '%s.wsgi.application' % SITE_NAME
 # END WSGI CONFIGURATION
 
-# AWS CONFIGURATION
 
+# AWS CONFIGURATION
 AWS_BUCKET_NAME = environ.get('RF_S3_BUCKET', 'raster-foundry-test-uploads-67dc48c70b3bcf89eab78dbf5cf7900')  # NOQA
-AWS_PROFILE = environ.get('AWS_PROFILE', 'rf-dev')
 AWS_SQS_QUEUE = environ.get('RF_SQS_QUEUE', 'TestQueue')
-AWS_SQS_REGION = environ.get('RF_SQS_REGION', 'us-east-1')
```

Comment:
Had to rebase because changes were made upstream to `process.py`. Actually fix should be in https://github.com/azavea/raster-foundry/commit/242ed5008eebb8efc218de1afa8a36bd501c85a6.